cmake/Modules/FindIlmBase.cmake

```diff
@@ -92,6 +92,7 @@ FIND_PATH(ILMBASE_INCLUDE_DIR
     IlmBaseConfig.h
   HINTS
     ${_ilmbase_SEARCH_DIRS}
+	"$ENV{ILMBASE_INC_DIR}"
```

Comment:
Its not clear to me why you added this, since ILMBASE_INC_DIR doest appear to be used as an env variable anywhere else.

Reply:
Yep, that's true, but it was the easiest solution to help to cmake to find the include dir. We use it here:
https://github.com/haikuports/haikuports/blob/4ab5e634c7fcab2bff7a2265204a2ecd4e9e31b1/media-libs/alembic/alembic-1.7.12.recipe#L114
Probably there is better way to accomplish this, feel free to recommend a better way.

Reply:
You can pass these variables into cmake:

-DILMBASE_INCLUDE_DIR=/usr/include/OpenEXR2  -DILMBASE_ROOT=/usr/lib64/OpenEXR2